modules/help_editor/jsx/help_editor_helper.js

```diff
@@ -0,0 +1,39 @@
+class EditContent extends React.Component {
+
+  render() {
+    function removeDiv(event) {
+      document.getElementById("preview").remove();
+      event.preventDefault();
+    }
+    const date = this.props.MyDate;
+    return (
+      <div>
+	<pre>
+          <h3>{this.props.Title}</h3>
+          <Markdown content={this.props.Content} />
+          <hr />
+          <p>Last updated: {date.getFullYear()}-{date.getMonth() + 1}-{date.getDate()} {date.getHours()}:{date.getMinutes()}:{date.getSeconds()}</p>
+	</pre>
+	<button className="btn btn-default" id="helpclose" onClick={removeDiv}>Close</button>
+      </div>
+    );
+  }
+
+}
+$(document).ready(function() {
```

Comment:
As @ZainVirani said, it's weird to have this mixture of jquery and React in one file. Why was this moved out of the other file?

Reply:
I said this??

Reply:
Oops.. I guess my eyes didn't read past "za". I was referring to the comment on line 1.